panel/viewable.py

```diff
@@ -326,7 +327,7 @@ def _on_stdout(self, ref: str, stdout: Any) -> None:
     def servable(
         self, title: Optional[str] = None, location: bool | 'Location' = True,
         area: str = 'main', target: Optional[str] = None
-    ) -> 'ServableMixin':
+    ) -> Self:
```

Comment:
As far as I can tell, mypy doesn't support type checking this yet (pyright does though).

Reply:
Okay, so I should use:

```python
_Self = TypeVar('_Self', bound='ServableMixin')
```

for now?

Reply:
Yes, sadly that's the only way for now. mypy seems quite behind the curve when it comes to implementing new PEPs.